src/api/app/components/workflow_run_filter_component.rb

```diff
@@ -1,17 +1,16 @@
 class WorkflowRunFilterComponent < ApplicationComponent
-  def initialize(token:, selected_filter:, finder:)
+  def initialize(token:, selected_filter:)
     super
 
-    @count = workflow_runs_count(finder)
+    @count = {
+      'success' => WorkflowRun.success.count,
```

Comment:
This would be all of the `WorkflowRun` in the database no?

Reply:
No, this would be only the 'success' ones, the failing and the running are not included here

Reply:
Yeah but the whole UI is about the ones of the token, not all of the ones in the database no?

Reply:
should be addressed now by https://github.com/openSUSE/open-build-service/pull/15950/commits/bcbf780c63ce276dc8d53b8b47057de760d14f82